codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocketClientHandshaker.java

```diff
@@ -273,7 +272,7 @@ public final void finishHandshake(Channel channel, FullHttpResponse response) {
             // Delay the removal of the decoder so the user can setup the pipeline if needed to handle
             // WebSocketFrame messages.
             // See https://github.com/netty/netty/issues/4533
-            channel.eventLoop().execute(new OneTimeTask() {
+            channel.eventLoop().execute(new Runnable() {
```

Comment:
@nmittler I could but Im not sure we should as basically the hack that we had with OneTimeTask is not needed anymore because we not use a queue that is based on a linked node structure. So this is related and so I think it should stay here.